### PR TITLE
sicer: new package

### PR DESCRIPTION
**Pull request by rmsds**

### CODE REVIEW

**Comment by rmsds** [comment]
See #205 instead.

**Comment by rmsds** [comment]
@alalazo would like your insight on why this one fails (thinks things changed in the list of tested core packages).

File: var/spack/repos/builtin/packages/sicer/package.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        install_tree('SICER', prefix.SICER)
+
+        # replacing a placeholder path in all the scripts
+        filter_file('/home/data/SICER1\.1', prefix,
+                    *(find(prefix.SICER, '*.sh', recursive=True)),
+                    backup=False)
+
+        # one of the scripts is not executable by default
+        chmod = which('chmod')
+        chmod('a+x', os.path.join(prefix.SICER, 'SICER.sh'))
```
Comment (nrichart) [inline var/spack/repos/builtin/packages/sicer/package.py:57]: to make it portable
```
import os
import stat

st = os.stat('somefile')
os.chmod('somefile', st.st_mode | stat.S_IEXEC)
```

**Review by nrichart — CHANGES_REQUESTED** [review]

File: var/spack/repos/builtin/packages/sicer/package.py (hunk 1 above)
Comment (rmsds) [reply var/spack/repos/builtin/packages/sicer/package.py:57]: I went with the spack builtin `set_executable()`.

I found it when investigating why stat didn't work, stat was actually overridden by a spack builtin within the class.

**Pull request closed without merging**